docs/test_configuration.rst

```diff
@@ -14,35 +12,21 @@ Since there are `multiple ways to configure pytest`_, we defined the following
 scheme:
 
 * Main *pytest config file* `pytest.ini`_ is committed in the root directory
-  of ``usmqe-tests`` repository. This file contains pytest configuration
-  and *default values* for most important USM QE configuration options. Under
-  normal circumstances (configuring ``usmqe-tests`` before test run) one would
-  not need to change this file at all, because any default value specified here
-  can be reconfigured in USM QE config file (with the exception of
-  ``usm_config`` and ``usm_inventory`` options).
-
-* *USM QE config file* is expected to contain actual configuration. See an
-  example in `conf/example_usm.ini`_, while the actual path of this file is
-  configured in ``usm_config`` option in main ``pytest.ini`` file. Any option
-  specified there overrides the default from main ``pytest.ini``. This is the
-  file one is supposed to create and change as needed. You need to provide
-  all important config values in this file to be able to run the tests.
-
-* Ansible *host inventory file* (see an example in `conf/example_usm.hosts`_),
-  which is used both by ansible and by USM QE inventory module to organize
-  machines into groups by it's role in test cluster. Actual path of this file
-  is configured in ``usm_inventory`` option in main ``pytest.ini`` file.
-
-* Moreover ad hoc reconfiguration of any USM QE option is possible via pytest
-  command line option ``--override-ini``. See an example how to use different
-  *host inventory file* for a particular test run:
-
-  .. code-block:: console
+  of ``usmqe-tests`` repository. This file contains pytest configuration.
 
-      $ py.test -o=usm_inventory=conf/mbukatov01.hosts usmqe_tests/foo/bar
+* *YAML configuration files* are expected to contain actual configuration.
+  `conf/main.yaml`_ should contain list of other configuration files that are
+  used as test configuration. There should be linked `conf/defaults.yaml`_ at
+  first place. After this there can be more configuration files that are
+  specific to testing enviroment (for more see example configuration
+  `conf/usm_example.yaml`_). `conf/defaults.yaml`_ file should contain test
+  configuration that is not environment specific.
 
-  This is usefull for test runs started by hand during test development or
-  debugging.
+* Ansible *host inventory file* (see an example in `conf/usm_example.hosts`_),
+  which is used both by ansible and by USM QE inventory module to organize
+  machines into groups by it's role in test cluster. Actual path of this file
+  is configured in one of the ``YAML configuration files``
```

Comment:
Isn't it configured in mail.yml file?

Reply:
It is now but it can be also configured in different one. The information that it is in main.yaml is specified on next line. As part of the sentence is:
```
(see `conf/main.yaml`_).
```

Reply:
ok